packages/synapse-interface/components/_Transaction/_Transaction.tsx

```diff
@@ -32,6 +32,7 @@ interface _TransactionProps {
   currentTime: number
   kappa?: string
   isStoredComplete: boolean
+  isStoredReverted: boolean
```

Comment:
Starting to have a few possible txn states to manage here, which make all the checkStatus & flow control logic complicated to read. What do you think about coalescing all these into a new `status` field instead of `isStoredComplete` or `isStoredReverted`?

A given transaction has a handful of statuses:
- Initiated (may not need this, but it does help follow a txn instead of `null`)
- Pending
- Complete
- Reverted

Reply:
Sounds good to me! I'll implement a status field to make flow control logic easier to parse. Thanks!

Reply:
Is this https://github.com/synapsecns/sanguine/pull/2080/files in the direction you're thinking of? Want to make sure I'm actually making things more legible